backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtApplicationConfiguration.java

```diff
@@ -58,6 +58,8 @@
 	/* Whether openURI will open page in new tab. By default it will, however if may be blocked by popup blocker. */
 	/* To prevent the page from being blocked you can redirect to the new page. However this will exit your game.  */
 	public boolean openURLInNewWindow = true;
+	/** whether to use the accelerometer. default: true **/
+	public boolean useAccelerometer = true;

```

Comment:
I know that on all other platforms this is `true`.
I didn't try but will this not cause every gwt game to ask for accelerometer permission on startup?

If so I would set this to `false`.

Reply:
Currently, there is no visual pop up in any browser. The default is allowed in all browsers.